obspy/io/rg16/__init__.py

```diff
@@ -89,16 +63,6 @@
 in a stream with seed compliant channel codes with orientations Z, N, E.
 
 >>> st = obspy.read(filename, contacts_north=True)
-
-Reading the initial headers
----------------------------
-The initial headers at the beginning of the rg16 file can be read
-separately using the specific function
-:func:`obspy.io.rg16.core._read_initial_headers` in the module
-:mod:`obspy.io.rg16`
-
->>> from obspy.io.rg16.core import _read_initial_headers
->>> initial_headers = _read_initial_headers(filename)
```

Comment:
Hmm. @thefroid this should maybe stay in, as this is really functionality that the user can not achieve with `core` high level routines.

Reply:
In fact, the information contained in the initial headers can be retrieved calling the read function with parameter details set to True. It creates a rg16 object containing information from initial headers and trace headers.

Reply:
Ah I see, alright then! Is unpacking these "details" very costly? Otherwise it might be worth a thought to unpack those details by default and use that switch to opt out for speed critical scenarios?

Reply:
Ok, then it's fine as it is now. :+1: